deploy/operator.yaml

```diff
@@ -131,13 +135,15 @@ spec:
             - name: NODE_NETWORK_STATE_REFRESH_INTERVAL
               valueFrom:
                 configMapKeyRef:
+                  optional: true
```

Comment:
You need to mount CM as a volume here too (this is worker DS, the one you changed is the master)